CHANGELOG.md

```diff
@@ -1043,4 +1057,7 @@ Major updates to Terrascan and the underlying architecture including:
 \* *This Changelog was automatically generated by [github_changelog_generator](https://github.com/github-changelog-generator/github-changelog-generator)*
 
 
+\* *This Changelog was automatically generated by [github_changelog_generator](https://github.com/github-changelog-generator/github-changelog-generator)*
```

Comment:
Can we somehow skip this credit link as it gets repeated multiple times?

Reply:
removed that repeated credits line manually, I will look for a command to skip